### PR TITLE
test: add test cases for taos shell restful

### PR DESCRIPTION
**Pull request by zhaoyanggh**

### CODE REVIEW

**Review by dingbo8128 — APPROVED** [review]
LGTM

File: tests/develop-test/0-others/taos_shell.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        self.checkresult("drop database if exists test", "Update OK")
+        self.checkresult("create database if not exists test", "Update OK")
+        self.checkresult("create stable test.stb (ts timestamp, c1 nchar(8), c2 double, c3 int) tags (t1 int)", "Update OK")
+        self.checkresult("create table test.tb1 using test.stb tags (1)", "Update OK")
+        self.checkresult("create table test.tb2 using test.stb tags (2)", "Update OK")
+        self.checkresult("select tbname from test.stb", "Query OK, 2 row(s) in set")
+        self.checkresult("insert into test.tb1 values (now, 'beijing', 1.23, 18)", "Update OK")
+        self.checkresult("insert into test.tb1 values (now, 'beijing', 1.23, 18)", "Update OK")
+        self.checkresult("insert into test.tb2 values (now, 'beijing', 1.23, 18)", "Update OK")
+        self.checkresult("insert into test.tb2 values (now, 'beijing', 1.23, 18)", "Update OK")
```
Comment (sangshuduo) [inline tests/develop-test/0-others/taos_shell.py:81]: suggestion: insert 100+ records and then query all records to check the record count is correct and only first 100 records output.

**Review by sangshuduo — COMMENTED** [review]
see my comment

File: tests/develop-test/0-others/taos_shell.py (hunk 1 above)
Comment (zhaoyanggh) [reply tests/develop-test/0-others/taos_shell.py:81]: Ok, done

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**